documentation/modules/exploit/linux/http/librenms_cmd_injection.md

```diff
@@ -0,0 +1,64 @@
+## Description
+  
+  In LibreNMS `v1.46` and below, there exists a command injection vulnerability in `capture.inc.php`.
+  The vulnerable functionality is intended to run a command such as `snmpwalk` and save the output as
+  a file. The `community` parameter is an unsanitized parameter retrieved through a POST request to `addhost`,
+  and it is used to build the command that is executed in the `capture.inc.php` functionality. The final command
+  is passed to the `popen()` function, which results in execution of arbitrary code.
+
+  This module has been tested on LibreNMS `v1.46` and `v1.45`.
+
+## Vulnerable Application
+
+  A vulnerable version of LibreNMS for Ubuntu can be installed using the instructions [here](https://docs.librenms.org/Installation/Installation-Ubuntu-1804-Apache/).
```

Comment:
It looks like there's also some pre-built OVA virtual appliances which may be worth testing and mentioning here:

* https://github.com/librenms/packer-builds/releases/tag/1.46

Reply:
Thanks, I'll list that!